redisinsight/ui/src/pages/rdi/pipeline/components/navigation/Navigation.tsx

```diff
@@ -8,6 +8,7 @@ import Download from 'uiSrc/pages/rdi/pipeline/components/download/Download'
 import JobsTree from 'uiSrc/pages/rdi/pipeline/components/jobs-tree'
 import RefreshPipelinePopover from 'uiSrc/pages/rdi/pipeline/components/refresh-pipeline-popover'
 import Tab from 'uiSrc/pages/rdi/pipeline/components/tab'
+import Upload from 'uiSrc/pages/rdi/pipeline/components/upload/Upload'
```

Comment:
Maybe better to call it UploadPopover? just a thought, not critical

Reply:
how about UploadModal instead? It's different from the existing popover that the Refresh functionality uses.